tests/testsuite/script.rs

```diff
@@ -1886,3 +1886,54 @@ CARGO_MANIFEST_PATH: [ROOT]/foo/script.rs
 "#]])
         .run();
 }
+
+#[cargo_test(nightly, reason = "edition2024 hasn't hit stable yet")]
```

Comment:
This test doesn't need nightly.  I know this is copied but I'm prepping a PR to remove it from the rest

Reply:
I was wondering about that but decided for consistency, removed now.